scripts/roboclawtest.py

```diff
@@ -9,17 +9,23 @@
 import sys
 from os import path
 # need to add the roboclaw.py file in the path
-sys.path.append(path.join(path.expanduser('~'), 'osr_ws/src/osr-rover-code/ROS/osr_control/osr_control'))
+sys.path.append(path.join(path.expanduser('~'), 'osr2_ws/src/osr-rover-code/ROS/osr_control/osr_control'))
 from roboclaw import Roboclaw
 
 if __name__ == "__main__":
     
     address = int(sys.argv[1]) 
-    roboclaw = Roboclaw("/dev/serial1", 115200)
-    try:
-        assert roboclaw.Open() == 1
-    except AssertionError as e:
-        raise e("Could not open comport /dev/serial1, make sure it has the correct permissions and is available")
-
-    print(roboclaw.ReadVersion(address))
-    print(roboclaw.ReadEncM1(address))
+    roboclaw0 = Roboclaw("/dev/serial0", 115200)
+    roboclaw1 = Roboclaw("/dev/serial1", 115200)
```

Comment:
also, why test both ports?

Reply:
Purely because the ports are a mess and inevitably people will get it to work on the wrong port. This will let them know that that's the case.